docs/tutorial/batch-minting.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Batch-minting NFTs on Ethereum 🚧
+description: This tutorial addresses some techniques in batch-minting big numbers of Ethereum non-fungible tokens (NFTs) as well as common idioms of uploading files and metadata on to IPFS/Filecoin via NFT.storage.
+issueUrl: https://github.com/protocol/nft-website/issues/253
+related: 
+  'Lazy minting': https://nftschool.dev/tutorial/lazy-minting/
+---
+
+ # Batch-minting NFTs on Ethereum
+
+ This tutorial addresses some techniques in batch-minting big numbers of Ethereum non-fungible tokens (NFTs) as well as common idioms of uploading files and metadata on to IPFS/Filecoin via NFT.storage.
+
+See also: [Lazy minting](./lazy-minting.md)
+
+Quite often,  NFT projects consist of hundreds if not thousands of files and metadata starting their lives out on a computer file system, waiting nervously to be part of a mint.
+
+There are cases in which it is desirable to mint many NFTs all at once instead of doing each of them in a [just-in-time](./lazy-minting.md) fashion. Large single drops are often for more than just art, including scenarios like large PFP (profile picture) drops, event tickets, game packs, and etc.
+
+Some challenges that lie in minting a bulk of NFTs include high gas fees on the Ethereum network, synchronizing between mint and upload failures and recovering from them, and performance of uploading files to NFT.storage, etc.
+
+## Batch-uploading NFT assets
+
+Here is an overview of strategies to upload to NFT.storage in bulk.
+
+### Upload files iteratively
+
+The most straightforward, yet more involved way to upload files and metadata to NFT.storage is by iterating over all of them and calling [`NFTStorage.store`](https://nftstorage.github.io/nft.storage/client/classes/lib.NFTStorage.html#store) one-by-one. Being simple as it is, that means you are responsible for properly handling errors that might occur for each successive request and synchronize it with the minting process.
```

Comment:
```suggestion
The most straightforward, yet more involved way to upload files and metadata to NFT.storage with code is by iterating over all of them and calling [`NFTStorage.store`](https://nftstorage.github.io/nft.storage/client/classes/lib.NFTStorage.html#store) one-by-one. While this seems straightforward on paper, it can create more complexity for your application. You are responsible for properly handling errors that might occur for each successive request and synchronize it with the minting process.
```